pkg/storage/mto_milestone.go

```diff
@@ -90,6 +90,16 @@ func MTOMilestoneUpdate(np sqlutils.NamedPreparer, _ *zap.Logger, MTOMilestone *
 	return returned, nil
 }
 
+// MTOMilestoneDelete deletes an MTOMilestone in the database
+func MTOMilestoneDelete(np sqlutils.NamedPreparer, _ *zap.Logger, milestoneID uuid.UUID) error {
```

Comment:
To work with change history, we need to set a  Session Variable for the current user.  This comes from the Plan Collaborator delete method
	err := setCurrentSessionUserVariable(tx, userID)
	if err != nil {
		return nil, err
	}

Reply:
This has been updated in 53a587fe7e2caeacf386dd75fb0c92cc6cece6a0 -- lemme know if that looks right to you, but I think it's all working as expected 👀